<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1538</width>
    <height>927</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout_parametry">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="parametry_main">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>700</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>700</height>
         </size>
        </property>
        <property name="title">
         <string>Parametry symulacji</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <widget class="QGroupBox" name="box_arx">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>331</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>ARX</string>
         </property>
         <widget class="QLineEdit" name="arxA_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>30</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_arxA">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>30</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Współczynniki A</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="arxB_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="opoznienie_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>110</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_arxB">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>70</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Współczynniki B</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_arxOpoznienie">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>110</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Opoźnienie</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="box_pid">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>331</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>PID</string>
         </property>
         <widget class="QLineEdit" name="PIDwzmocnienie_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>30</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="PIDTi_input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="PIDTd_input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>110</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_PIDwzmocnienie">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>30</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Wzmocnienie</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_PIDTi">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>131</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Stała całkowania Ti</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_PIDTd">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>161</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Stała różniczkowania Td</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="box_gen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>300</y>
           <width>331</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>Generator</string>
         </property>
         <widget class="QLineEdit" name="GenAmp_input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>30</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="GenT_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="GenFill_Input">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>110</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_GenAmp">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Amplituda</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_GenT">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>70</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>T</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_GenFill">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>110</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Wypełnienie</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="box_add">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>440</y>
           <width>331</width>
           <height>131</height>
          </rect>
         </property>
         <property name="title">
          <string>Dodatkowe</string>
         </property>
         <widget class="QComboBox" name="genType_Box">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>30</y>
            <width>82</width>
            <height>28</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Square</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Skok</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_genType">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Typ generatora</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="interwal_Input">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_interwal">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>70</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Interwał (s)</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="zaklocenia_button">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>100</y>
            <width>112</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>Zakłócenia</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="start_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>580</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stop_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>610</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QPushButton" name="reset_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>650</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
        <widget class="QPushButton" name="save_button">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>590</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
        <widget class="QPushButton" name="load_button">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>640</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Wczytaj</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="layout_wykres1"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_wykres2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
